docs/user/installation_and_configuration.rst

```diff
@@ -459,3 +475,12 @@ is printed out when you start GeoServer::
     ----------------------------------
     - GEOSERVER_DATA_DIR: /opt/devel/install/geoserver-data-dir
     ----------------------------------
+
+Note about Apache Commons Collections
+-------------------------------------
+
+Version 3.2.1 and earlier of the Apache Commons Collections library have a CVSS 10.0 vulnerability.  Read more `here
+https://commons.apache.org/proper/commons-collections/security-reports.html`__.
+
+Note that Accumulo 1.6.5 is the first version of Accumulo which addresses this security concern.
```

Comment:
Nope, but versions of Accumulo seem to be 'binary' compatible.

Would it make sense to spell out the implications?  Or would that confuse matters?